msteams-platform/toolkit/add-single-sign-on.md

```diff
@@ -125,236 +127,295 @@ SSO command handler `ProfileSsoCommandHandler` uses an Azure AD token to call Mi
+
+       let photoUrl = "";
+       try {
+         const photo = await graphClient.api("/me/photo/$value").get();
+         const arrayBuffer = await photo.arrayBuffer();
+         const buffer = Buffer.from(arrayBuffer, "binary");
+         photoUrl = "data:image/png;base64," + buffer.toString("base64");
+       } catch {
+         // Could not fetch photo from user's profile, return empty string as placeholder.
+       }
+       if (photoUrl) {
+         const photoMessage = {
+           type: ActivityTypes.Message,
+           text: "This is your photo:",
+           attachments: [
+             {
+               name: "photo.png",
+               contentType: "image/png",
+               contentUrl: photoUrl,
+             },
+           ],
+         };
+         return photoMessage;
+       } else {
+         return "Could not retrieve your photo from Microsoft Graph. Please make sure you have uploaded your photo.";
+       }
+     }
+   }
+
+   module.exports = {
+     PhotoSsoCommandHandler,
+   };
+   ```
 
-    ```javascript
-    // for JavaScript:
-    const { ActivityTypes } = require("botbuilder");
-    require("isomorphic-fetch");
-    const { createMicrosoftGraphClient, TeamsFx } = require("@microsoft/teamsfx");
-
-    class PhotoSsoCommandHandler {
-        triggerPatterns = "photo";
-
-        async handleCommandReceived(context, message, tokenResponse) {
-            await context.sendActivity("Retrieving user information from Microsoft Graph ...");
-
-            const teamsfx = new TeamsFx().setSsoToken(tokenResponse.ssoToken);
-
-            const graphClient = createMicrosoftGraphClient(teamsfx, ["User.Read"]);
-        
-            let photoUrl = "";
-            try {
-                const photo = await graphClient.api("/me/photo/$value").get();
-                const arrayBuffer = await photo.arrayBuffer();
-                const buffer=Buffer.from(arrayBuffer, 'binary');
-                photoUrl = "data:image/png;base64," + buffer.toString("base64");
-            } catch {
-            // Could not fetch photo from user's profile, return empty string as placeholder.
-            }
-            if (photoUrl) {
-                const photoMessage = {
-                    type: ActivityTypes.Message, 
-                    text: 'This is your photo:', 
-                    attachments: [
-                        {
-                            name: 'photo.png',
-                            contentType: 'image/png',
-                            contentUrl: photoUrl
-                        }
-                    ]
-                };
-                return photoMessage;
-            } else {
-                return "Could not retrieve your photo from Microsoft Graph. Please make sure you have uploaded your photo.";
-            }
-        }
-    }
+1. Add `PhotoSsoCommandHandler` instance to `ssoCommands` array in `bot/src/internal/initialize.ts`:
```

Comment:
As per guidelines to write a folder path: In command syntax, use path to represent only the folder portion of the full path: copy [drive:][path]filename.

Request you to check and update the file path.

Reply:
As discussed with the PM, the PM explained that we can't add a dive path as it depends on where the user installs the package. He also confirmed that the folder structure remains the same.

Change not required.